### ISSUE TITLE
Contributing.md might be missing steps

### ISSUE THREAD

**Issue by aderiyenko** [DataDog/dd-trace-php#756]
Trying to setup local environment to launch integrations tests of https://github.com/DataDog/dd-trace-php/pull/755 locally. 

On `docker-compose up` in one of docker images build (specifically `7.1-centos-compiled`) getting:

Step 11/15 : ADD build/packages /packages
ERROR: Service '7.1-centos-compiled' failed to build: ADD failed: stat /var/lib/docker/tmp/docker-builder016978041/build/packages: no such file or directory

There is no `build` directory in the project root at this point.

Not sure if I had to run `local-build.sh` before that - tried to do so and got `cc1: error: unrecognized command line option "-std=gnu11"` on PHP 5.4 extension build. Even if I add 

`  docker run --rm -w /var/app -v $(pwd):/var/app datadog/docker-library:ddtrace_centos_6_php_5_4 \
    yum -y install curl-devel &&
    make all CFLAGS="-O2 -std=c99 -Wall -Wextra -Wextra" ECHO_ARG="-e" &&
    cp tmp/build_extension/.libs/ddtrace.so extensions/ddtrace-${PHP_API}.so
`
to the build instruction, the .so is built, but then it fails on the 5.6 extension modified build 
`  docker run --user=root --rm -w /var/app -v $(pwd):/var/app circleci/php:${PHP_VERSION}${SUFFIX} \
    apt-get update &&
    apt-get install -y libcurl4-openssl-dev &&
    make all CFLAGS="-O2 -std=c99 -Wall -Wextra -Wextra" &&
    cp tmp/build_extension/.libs/ddtrace.so extensions/ddtrace-${PHP_API}${SUFFIX}.so`

with:

`./local-build.sh: line 17: apt-get: command not found`

**Comment by labbati** [comment]
Very good point @aderiyenko! We definitely need to get back to our contributing pages and bring up to date with current process. We will work on this asap.

**Comment by aderiyenko** [comment]
Thanks! Right now the blocking issue for me is:

seems that all integration tests code has hardcoded `http://localhost:9999` endpoints. Which is not correct when you launch the suite from the container environment as stated in the doc. In this case localhost would be the container itself, but the web server on 9999 port is running in another container exposing his port to the host machine. So the correct hostname would be `host.docker.internal:9999` in this case. 

I will not change the hostname in the code as it runs fine in circleCI environment on your side, but wanted to address this somehow in my environment. 

I tried to set `network_mode: host` in the base section of docker-compose.yaml, but it did not work, probably because I'm on mac. Still seeing localhost:9999 connection failures from the suite

PS. I'm able to connect to http://localhost:9999 with a browser on my host machine, so the dependency container with the web server is actually running

**Comment by aderiyenko** [comment]
so after further debugging I realised I was wrong in my assumption that the 0.0.0.0:9999 web server is launched outside of docker container. In fact, it could be launched from anywhere as it's done in `tests/WebServer.php` and it depends on where one runs the suite from.

Apparently, looking at my message above I was able to launch the web server process fine on the host machine, but not in the container. 

If I increase `usleep(...)` interval after launching the web server, I can connect to it with telnet inside docker container as localhost:9999, but seems the process is killed at some point after `usleep` is done. 

How can I debug if the error log file passed with `-d` options is not created in the container?
